cmd/backup/config.go

```diff
@@ -28,6 +28,7 @@ type Config struct {
 	AwsSecretAccessKey            string        `split_words:"true"`
 	AwsSecretAccessKeyFile        string        `split_words:"true"`
 	AwsIamRoleEndpoint            string        `split_words:"true"`
+	AwsDefaultPartSize            int64         `split_words:"true" default:"16"`
```

Comment:
Considering the currently used value on `main` is the "zero value" for `int64` I would think 0 is the correct default here, as it does exactly what is done before.